config/config.json

```diff
@@ -45,14 +45,17 @@
       "enabled": true
     },
     "list": {
-      "edu.harvard.dce.paella.hideableExtendedTabAdapterPlugin": {
+      "edu.harvard.dce.paella.handoutDownloadPlugin" : {
         "enabled":true
       },
+      "edu.harvard.dce.paella.hideableExtendedTabAdapterPlugin": {
+        "enabled":false
+      },
       "es.upv.paella.matterhorn.downloadsPlugin" : {
          "enabled":false
       },
       "edu.harvard.dce.paella.tabBarHandoutDownlaodPlugin" : {
```

Comment:
that's a typo that's also part of the HUDCE plugin name from the last commit. If I fix that spelling, I'll need to fix it through that plugin too. Are you Ok if we just create a new ticket to clean that up?